~This is still failing on CI apparently with the Chromatic errors I mentioned on Slack, e.g.~
```
chromatic info Uploaded your build, verifying 
[snip]
chromatic info 52/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 51/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 50/52 snapshots remain to test. (0 changes, 0 errors) 
chromatic info 49/52 snapshots remain to test. (1 change, 0 errors) 
chromatic info 48/52 snapshots remain to test. (2 changes, 0 errors) 
chromatic info 47/52 snapshots remain to test. (2 changes, 0 errors) 
Sending interrupt signal to process
Terminated
Makefile:10: recipe for target 'developmentTests' failed
make: *** [developmentTests] Terminated
script returned exit code 143
````

**Update**: it's nothing to do with the above as Drew pointed out, red herring. The e2e tests were failing on services like Persian, Serbian, etc. because we weren't checking the media block exists before trying to look for a caption. Fixed in c3c3ea9

--- a/cypress/integration/pages/articles/tests.js
+++ b/cypress/integration/pages/articles/tests.js
@@ -197,24 +197,27 @@ export const testsThatFollowSmokeTestConfig = ({
           it('should have a visible caption beneath a mediaplayer', () => {
             cy.request(`${config[service].pageTypes.articles.path}.json`).then(
               ({ body }) => {
-                const mediaData = getBlockData('video', body);
-                const captionBlock = getBlockByType(
-                  mediaData.model.blocks,
-                  'caption',
-                );
-                const {
-                  text,
-                } = captionBlock.model.blocks[0].model.blocks[0].model;
+                const media = getBlockData('video', body);
+                if (media) {
+                  const captionBlock = getBlockByType(
+                    media.model.blocks,
+                    'caption',
+                  );
 
-                if (captionBlock) {
-                  cy.get('figcaption')
-                    .eq(1)
-                    .within(() => {
-                      cy.get('p')
-                        .eq(0)
-                        .should('be.visible')
-                        .should('contain', text);
-                    });
+                  if (captionBlock) {
+                    const {
+                      text,
+                    } = captionBlock.model.blocks[0].model.blocks[0].model;
+
+                    cy.get('figcaption')
+                      .eq(1)
+                      .within(() => {
+                        cy.get('p')
+                          .eq(0)
+                          .should('be.visible')
+                          .should('contain', text);
+                      });
+                  }
                 }
               },
             );

